From 1937698c25fb9f066dba7b48ad4731927e3e74da Mon Sep 17 00:00:00 2001
From: Richard W.M. Jones <rjones@redhat.com>
Date: Tue, 2 Nov 2010 20:33:06 +0000
Subject: [PATCH] daemon: Tolerate failure of blockdev --rereadpt after sfdisk.

See commit 840536ea5a0568296dfd3e483442c76b93c5a949
and commit 956fc5a3feacc970ea763697bf28fb686c875408.
---
 daemon/sfdisk.c |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/daemon/sfdisk.c b/daemon/sfdisk.c
index bf62c1d..ec77465 100644
--- a/daemon/sfdisk.c
+++ b/daemon/sfdisk.c
@@ -101,8 +101,7 @@ sfdisk (const char *device, int n, int cyls, int heads, int sectors,
    * other component.  In any case, reread the partition table
    * unconditionally here.
    */
-  if (do_blockdev_rereadpt (device) == -1)
-    return -1;
+  (void) command (NULL, NULL, "blockdev", "--rereadpt", NULL);
 
   udev_settle ();
 
-- 
1.7.3.1

